### PR TITLE
Return telemetry in teams request. Index (user,timestamp) for telemetry.

### LINKED ISSUE #164: Make latest telemetry queries more efficient
We hit a scalability problem with latest queries for telemetry. @prattmic improved the db query substantially, which will help us survive. We can use the now consistent cache to store the result of "latest" queries, like latest telemetry for each user.

### PR DESCRIPTION
**Pull request by pmtischler**
Returning telemetry in the teams request makes fewer requests and
database queries.

Indexing (user,timestamp) should allow simultaneous user filtering,
timestamp filtering, and ordering by timestamp via the index without
having to post-filter and post-sort. This should make requests more
efficient.

Fixes #164

### CODE REVIEW

File: server/auvsi_suas/models/access_log_test.py [hunk 1]
```diff
@@ -104,36 +104,39 @@ def test_multi_user(self):
...
+                                       delta=delta)
+
+        logs = UasTelemetry.by_user(self.user1,
+                                    start_time=start,
+                                    end_time=start + delta * 10)
+        self.assertSequenceEqual(expect_logs, logs)
+
+        logs = UasTelemetry.by_user(self.user1, start_time=start + delta * 11)
+        self.assertSequenceEqual([], logs)
+        logs = UasTelemetry.by_user(self.user1, end_time=start - delta)
```
Comment (prattmic) [inline server/auvsi_suas/models/access_log_test.py]: You could get the same results with end_time=start

File: server/auvsi_suas/views/teams_test.py [hunk 2]
```diff
@@ -352,12 +366,12 @@ def test_id_ignored(self):
 
         self.assertEqual(expected, data['id'])
 
-    def test_active_ignored(self):
-        """active field ignored"""
+    def test_telemetry_ignored(self):
+        """telemetryfield ignored"""
```
Comment (prattmic) [inline server/auvsi_suas/views/teams_test.py]: telemetry field

Comment (pmtischler) [inline server/auvsi_suas/views/teams_test.py]: Done.

File: server/auvsi_suas/models/access_log_test.py (hunk 1 above)
Comment (pmtischler) [inline server/auvsi_suas/models/access_log_test.py]: Done.

**Comment by pmtischler** [comment]
I'm attempting to evaluate now. I pulled the data off of the interop laptop and uploaded to our Drive folder. Import went quick, but migration is taking a long time (still on the first, migration 18). Given how long this is taking, I'll upload the migrated form as well to save us time in future testing.

**Comment by pmtischler** [comment]
Test completed. Was able to look at admin interface and CPU stayed under 15% peak, with about ~10% average.

**Pull request merged**